src/tabs/tab-header-group.component.ts

```diff
@@ -86,6 +87,19 @@ export class TabHeaderGroup implements AfterContentInit, OnDestroy {
 	@Input() contentAfter: TemplateRef<any>;
 
 	@Input() contentBefore: TemplateRef<any>;
+
+		/**
```

Comment:
some weird tabbing going on here ...

```suggestion
	/**
```